pokerth.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 PokerTH GbR <webmaster@pokerth.net> -->
+<component type="desktop-application">
+ <id type="desktop">pokerth.desktop</id>
```

Comment:
As I have said to you many times, if you include the appstream file in the flathub repo because it doesn't exist upstream then it should be correct and not need "rename". Sone the file name, line 4 and 27 should be addressed.

This is not a new thing.

Reply:
Look at the patch. It is already part of the upstream repository. https://github.com/pokerth/pokerth/blob/master/pokerth.appdata.xml I just couldn't easily extract it from the commits.